<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>390</width>
    <height>310</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Free convert</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../Дизигн/icons/Mercury_Icon_Set/Mercury_Static_icons/PNG/24pix/B_stat24_navigation_refresh.png</normaloff>
    <normalon>../../Дизигн/icons/Mercury_Icon_Set/Mercury_Static_icons/PNG/24pix/B_stat24_navigation_refresh.png</normalon>
    <disabledoff>../../Дизигн/icons/Mercury_Icon_Set/Mercury_Static_icons/PNG/24pix/B_stat24_navigation_refresh.png</disabledoff>
    <disabledon>../../Дизигн/icons/Mercury_Icon_Set/Mercury_Static_icons/PNG/24pix/B_stat24_navigation_refresh.png</disabledon>
    <activeoff>../../Дизигн/icons/Mercury_Icon_Set/Mercury_Static_icons/PNG/24pix/B_stat24_navigation_refresh.png</activeoff>
    <activeon>../../Дизигн/icons/Mercury_Icon_Set/Mercury_Static_icons/PNG/24pix/B_stat24_navigation_refresh.png</activeon>
    <selectedoff>../../Дизигн/icons/Mercury_Icon_Set/Mercury_Static_icons/PNG/24pix/B_stat24_navigation_refresh.png</selectedoff>
    <selectedon>../../Дизигн/icons/Mercury_Icon_Set/Mercury_Static_icons/PNG/24pix/B_stat24_navigation_refresh.png</selectedon>../../Дизигн/icons/Mercury_Icon_Set/Mercury_Static_icons/PNG/24pix/B_stat24_navigation_refresh.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 39, 28);</string>
  </property>
  <property name="inputMethodHints">
   <set>Qt::ImhNone</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>501</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 170, 127);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>0</y>
       <width>421</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 87 28pt &quot;Segoe UI Black&quot;;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 170, 127), stop:1 rgba(0, 100, 127));</string>
     </property>
     <property name="text">
      <string>FREE CONVERT</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../Дизигн/icons/Mercury_Icon_Set/Mercury_Static_icons/PNG/48pix/W_stat48_navigation_refresh.png</pixmap>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="amount_line_in">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>400</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 130, 87);
color: rgb(255, 255, 255);
border: 2px;
border-radius: 10;
font: 25 20pt &quot;Segoe UI Black&quot;;

background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 170, 127), stop:1 rgba(0, 100, 127));</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>100</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="amount_line_out">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>400</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 130, 87);
color: rgb(255, 255, 255);
border: 2px;
border-radius: 10;
font: 25 20pt &quot;Segoe UI Black&quot;;

background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 170, 127), stop:1 rgba(0, 100, 127));</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>100</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="convert_btn">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>260</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 170, 127);
border-radius: 22;

background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 170, 127), stop:1 rgba(0, 100, 127));</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../Дизигн/icons/Mercury_Icon_Set/Mercury_Static_icons/PNG/48pix/W_stat48_navigation_arrow_right.png</normaloff>
      <normalon>../../Дизигн/icons/Mercury_Icon_Set/Mercury_Static_icons/PNG/48pix/W_stat48_navigation_arrow_down.png</normalon>../../Дизигн/icons/Mercury_Icon_Set/Mercury_Static_icons/PNG/48pix/W_stat48_navigation_arrow_right.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QListWidget" name="cur_line_in">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>140</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 130, 87);
color: rgb(255, 255, 255);
border: 2px;
border-radius: 10;
font: 25 20pt &quot;Segoe UI Black&quot;;

background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 170, 127), stop:1 rgba(0, 100, 127));</string>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContents</enum>
    </property>
    <property name="autoScroll">
     <bool>true</bool>
    </property>
    <property name="autoScrollMargin">
     <number>16</number>
    </property>
    <property name="tabKeyNavigation">
     <bool>false</bool>
    </property>
    <property name="showDropIndicator" stdset="0">
     <bool>true</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::NoSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectItems</enum>
    </property>
    <property name="textElideMode">
     <enum>Qt::ElideMiddle</enum>
    </property>
    <property name="flow">
     <enum>QListView::TopToBottom</enum>
    </property>
    <property name="resizeMode">
     <enum>QListView::Fixed</enum>
    </property>
    <property name="layoutMode">
     <enum>QListView::SinglePass</enum>
    </property>
    <property name="viewMode">
     <enum>QListView::ListMode</enum>
    </property>
    <property name="batchSize">
     <number>100</number>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="selectionRectVisible">
     <bool>false</bool>
    </property>
    <property name="currentRow">
     <number>0</number>
    </property>
    <item>
     <property name="text">
      <string>USD</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>EUR</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>RUB</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>GBP</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>JPY</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CHF</string>
     </property>
    </item>
   </widget>
   <widget class="QListWidget" name="cur_line_out">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>140</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 130, 87);
color: rgb(255, 255, 255);
border: 2px;
border-radius: 10;
font: 25 20pt &quot;Segoe UI Black&quot;;

background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 170, 127), stop:1 rgba(0, 100, 127));</string>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="showDropIndicator" stdset="0">
     <bool>true</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::NoSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectItems</enum>
    </property>
    <property name="textElideMode">
     <enum>Qt::ElideMiddle</enum>
    </property>
    <property name="flow">
     <enum>QListView::TopToBottom</enum>
    </property>
    <property name="resizeMode">
     <enum>QListView::Fixed</enum>
    </property>
    <property name="layoutMode">
     <enum>QListView::SinglePass</enum>
    </property>
    <property name="viewMode">
     <enum>QListView::ListMode</enum>
    </property>
    <property name="batchSize">
     <number>100</number>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="selectionRectVisible">
     <bool>false</bool>
    </property>
    <property name="currentRow">
     <number>-1</number>
    </property>
    <item>
     <property name="text">
      <string>USD</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>EUR</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>RUB</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>GBP</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>JPY</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CHF</string>
     </property>
    </item>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
